pkg/registry/core/service/storage/storage.go

```diff
@@ -127,6 +123,10 @@ func NewREST(
 	store.BeginCreate = genericStore.beginCreate
 	store.BeginUpdate = genericStore.beginUpdate
 
+	statusStore := *store
```

Comment:
or should we add only the afterDelete hook?

Reply:
yeah, we should not add the other hooks or they will be exercised, and who knows what consequences this can bring